src/components/CallView/shared/VideoVue.vue

```diff
@@ -48,8 +52,11 @@
 					:source="participantActorType"
 					:size="avatarSize"
 					disable-menu
-					disable-tooltip
-					:class="avatarClass" />
+					disable-tooltip />
+				<NcLoadingIcon v-if="isLoading"
```

Comment:
For me it makes more sense to integrate it into `AvatarWrapper` component, with `loading` prop, maybe?

Reply:
Sounds like a good idea 👍🏻